Add new Choreo integration (#1806)
DataDog/integrations-extras: choreo/README.md

# Agent Check: Choreo

## Overview

[Choreo][1] is a SaaS application development suite designed to accelerate the creation of digital experiences. You can use Choreo to build, deploy, monitor, and manage your cloud-native applications, allowing you to increase developer productivity and focus on innovation.

> This integration enables you to observe [Ballerina][2] components running in Choreo private data planes. The Choreo integration creates a dashboard that can be used to view application metrics, system metrics, and logs related to your Ballerina components. 

Reviewer: ```suggestion
This integration enables you to observe [Ballerina][2] components running in Choreo private data planes. The Choreo integration creates a dashboard with application metrics, system metrics, and logs related to your Ballerina components. 
```

Reviewer: Optional suggestion here to make this sentence more concise.

## Setup

### Installation

Setting up Datadog observability for your private data plane requires support from the WSO2 team. Reach out to your WSO2 Account Manager or [contact us here][3] so that the WSO2 team can set up the Datadog Agent in the cluster.

> 

Reviewer 2: Please update this to the new setup instructions - thanks!
